packages/ketcher-core/src/application/indigo.ts

```diff
@@ -200,12 +201,13 @@ export class Indigo {
   ): Promise<string> {
     const outputFormat = options?.outputFormat || 'png';
     const backgroundColor = options?.backgroundColor || '';
-
+    const bondThickness = options?.bondThickness || 1;
```

Comment:
I propose to create a constant for default value, which will be used across app :) 
And additional question. Here you have `1` default value, but somewhere below you have `2`. Why? :)

Reply:
propose to create a constant for default value - _Done_.
Regarding value `2` it is used just for testing